frontstage/templates/secure-messages/secure-messages-view.html

```diff
@@ -28,12 +16,56 @@
         <div class="grid__col col-4@m"></div>
         <div class="grid__col col-7@m pull-1@m">
         {% if label != 'SENT' %}
-            {% include 'secure-messages/message-form.html' %}
+            {% set message = {} if not message else message %}
+
+            {% if not message %}<h1 class="saturn">Create message</h1>{% endif %}
+
+            <div class="secure-message-form">
+                <form action="{{ url_for('secure_message_bp.create_message', ru_ref=ru_ref, survey=survey, case_id=case_id) }}" method="post">
+                    <input type="hidden" name="csrf_token" value="{{ csrf_token() }}"/>
+                    {{ form.thread_id }}
+
+                    {% if message %}
+                        {{ form.hidden_subject }}
+
+                    {% else %}
+                        <p>To: ONS Business Surveys team</p>
+                        {% if "subject" in errors %}
+                            <div class="panel panel--simple panel--error panel--spacious" id="subject-error">
+                                <p class="error-message">{{ errors['subject'][0] }}</p>
+                        {% endif %}
+                            <label class="venus" for="subject">Subject</label>
+                            {{ form.subject(id='secure-message-subject', class_='secure-message-form__subject input input--text') }}
+                        {% if "subject" in errors %}
+                            </div>
+                        {% endif %}
+                        <br />
+                    {% endif %}
+
+                    {% if "body" in errors %}
+                        <div class="panel panel--simple panel--error panel--spacious" id="body-error">
+                            <p class="error-message">{{ errors['body'][0] }}</p>
+                    {% endif %}
+                    {% if message %}
+                        <label class="venus" for="body">Reply</label>
+                    {% else %}
+                        <label class="venus" for="body">Message</label>
+                    {% endif %}
+                    {{ form.body(id='secure-message-body', class_='input input--textarea input--textarea-message', rows='10') }}
```

Comment:
Should this have something like maxlength='10000'?

Reply:
I only removed the included template and dropped the content of it into here.  If it's wrong here then it's wrong on master too and should have a branch taken off and fixed there.
I'm trying to reduce the scope of the changes in this branch because I'm deleting/changing quite a few files.